### ISSUE TITLE
Camera buttons not appearing ingame

### ISSUE THREAD

**Issue by Rqdu96** [Oplexitie/Cupcakes-Framework#1]
Title explains it all, the cameras button doesnt show up.

**Comment by Rqdu96** [comment]
Sorry i meant button*

**Comment by Oplexitie** [comment]
@Rqdu96 
Hello, after doing some testing on my side, I can't seem to reproduce this issue...
Could you please give me more information ?
Did this happen in an unmodified version of the framework ?
If yes, could you please tell me where you downloaded the Framework, and which version of Godot you're using ?
Could you also tell me how you triggered this issue ?

If this happened in a modified version, can you tell me what you modified ?

**Comment by Rqdu96** [comment]
I downloaded the framework from the official godot asset manager or wtv its called, my version of godot is 4.1. Note: i am using linux. I dont know if its an linux only issue.

**Comment by Rqdu96** [comment]
I accidently closed it ._.

**Comment by Rqdu96** [comment]
The node configuration warning says: "changing the z index of a control only affects the drawing order,not the input event handling order"

**Comment by Rqdu96** [comment]
I didnt modify anything tho.

**Comment by Oplexitie** [comment]
@Rqdu96
Ok, so I just replicated this issue on my Linux labtop.
And I figured out the problem, it's not a Linux issue, it's because our Linux labtop screen resolutions are smaller than the projects resolution. This can be fixed easily by going into ProjectSettings -> Display -> Window and in the Stretch section set the mode to Viewport.

![image](https://github.com/Oplexitie/Cupcakes-Framework/assets/72892666/24e725fb-0b2f-4a56-a9ce-f7ea058e39d3)

Thank you for informing of this issue, i'll push a fix as soon as I can.

As for the "changing the z index of a control only affects the drawing order,not the input event handling order" thing that's normal, it's just a warning meant to inform the user about the effects of changing a buttons layer.
